database/migrations/20220322155856-create-gm-products-table.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.createTable("gm_products", {
+      uuid: {
+        allowNull: false,
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4,
+      },
+      name: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      category: {
```

Comment:
@maheshmuttinti why do you need this `category` column?

Reply:
If the user wants to add the category to a particular product. we need this `category` column while adding product @rajeshn95. Am I right? or does anything here needs changes?